src/StreetNameRegistry.Migrator.StreetName/Infrastructure/Program.cs

```diff
@@ -69,6 +70,8 @@ public static async Task Main(string[] args)
                     {
                         try
                         {
+                            MigrationsHelper.Run(configuration.GetConnectionString("Sequence"));
```

Comment:
Won't be needed, since backoffice has it covered

Reply:
Removed.